SEED-platform/seed
# AH - Revise analysis permissions for creating columns

#### What's this PR do?
Restrict creation of new extra data columns during an analysis run to root owner and members.
If the analysis is run by a leaf member, and the columns do not exist, they will not be created. If they exist, they will be populated with the analysis results.

#### How should this be manually tested?
Import data in a hierarchy-enabled org. 
Create a leaf node user
As leaf node user, create an analysis. Verify that no columns were created on the org.
As a root node user (owner or member role), create an analysis. Verify that columns were created and populated.

eligible analyses:  eui, co2, eeej, BETTER.

#### What are the relevant tickets?
#4220

#### Screenshots (if appropriate)
